terminus-ui/select/src/select.component.spec.ts

```diff
@@ -52,13 +51,25 @@ import {
   getSelectInstance,
   getSelectTriggerElement,
   getToggleAllElement,
+  openSelect,
 } from '@terminus/ui/select/testing';
 import { getValidationMessageElement } from '@terminus/ui/validation-messages/testing';
+import { createComponent as createComponentInner } from '@terminus/ngx-tools/testing';
 
 import { TsSelectOptionComponent } from './option/option.component';
 import { TsSelectModule, TsSelectFormatFn, TsSelectOption } from './select.module';
 
 
+function createComponent<T>(component: Type<T>): ComponentFixture<T> {
+  const moduleImports = [
+    FormsModule,
+    ReactiveFormsModule,
+    TsSelectModule,
+    NoopAnimationsModule,
+  ];
+
+  return createComponentInner(component, undefined, moduleImports);
```

Comment:
Though this is confusing, it's at least contained.

Is there ever a scenario where the "outer" component needs to be used for testing?

Reply:
This is just a helper to make it easier in the test file. Since every usage of `createComponent` needs the module imports added I just created this helper function so that I didn't need to add the imports to every single test.

It returns the original function return so there is no outer component - it is exactly the same as using the standard `createComponent` except it passes in the needed imports.